client/python/armada_client/client.py

```diff
@@ -235,16 +273,41 @@ def create_job_request(
         )
 
     def create_job_request_item(
-        self, pod_spec: core_v1.PodSpec, priority: int = 1, **job_item_params
+        self,
+        pod_spec: core_v1.PodSpec,
+        priority: float = 1,
```

Comment:
```suggestion
        priority: float = 1.0,
```

nit pick.  Python probably doesn't even care about this.